images/dockerregistry/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#
+# This is the integrated OpenShift Origin Docker registry. It is configured to
+# publish metadata to OpenShift to provide automatic management of images on push.
+#
+# The standard name for this image is openshift/origin-docker-registry
+#
+FROM openshift/origin-base
+
+ADD bin/dockerregistry /usr/bin/dockerregistry
+
+COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
```

Comment:
I understand this is pre-existing ... but what sets this var?

Reply:
yeah i dunno, i don't see anything setting it.  weirdly the value is set a few lines lower in the dockerfile.  I'm not sure how this ever worked?

Reply:
(looks like things expect config.yaml to be in / )

Reply:
(and it looks like that's where docker dumps the file if the variable is undefined).

so i guess this is "ok" but it's not clear to me if it's intentionally working,  or accidentally.

anyway it's fine to leave it for this PR.